pom.xml

```diff
@@ -20,14 +20,14 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.netease.arctic</groupId>
-    <artifactId>arctic-parent</artifactId>
+    <groupId>com.netease.amoro</groupId>
+    <artifactId>amoro-parent</artifactId>
     <version>0.5.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>Arctic Project Parent</name>
+    <name>Amoro Project Parent</name>
     <description>Arctic is a LakeHouse management service for open architecture</description>
```

Comment:
It should be `Amoro` instead of `Arctic`

Reply:
yes, i will fix it.